.github/workflows/build.yml

```diff
@@ -46,12 +46,22 @@ jobs:
           repository: quarkusio/quarkus
           path: ./quarkus
 
-      - name: Quarkus integration
+      - name: Quarkus integration (JAR)
         working-directory: ./quarkus
         run: |
-          mvn -Dquickly
-          java -jar ../target/quarkus-app/quarkus-run.jar &
-          mvn clean verify -pl tcks/microprofile-lra -Dtcks -Dlra.coordinator.url=http://localhost:8080/lra-coordinator
-          kill %1
+#          mvn -Dquickly
+#          java -jar ../target/quarkus-app/quarkus-run.jar &
+#          mvn clean verify -pl tcks/microprofile-lra -Dtcks -Dlra.coordinator.url=http://localhost:8080/lra-coordinator
+#          kill %1
+
+      - name: Quarkus integration (image)
+        run: |
+          # build the image quay.io/jbosstm/lra-coordinator:1.0.0-SNAPSHOT
+          ./mvnw clean package -Dquarkus.container-image.build=true
```

Comment:
Nice work! Could we inject the image tag?

Reply:
No, this is intentionally using SNAPSHOT.